src/core/meshcop/joiner.cpp

```diff
@@ -143,6 +143,9 @@ otError Joiner::Start(const char *     aPskd,
     otLogInfoMeshCoP("Joiner starting");
 
     VerifyOrExit(mState == OT_JOINER_STATE_IDLE, error = OT_ERROR_BUSY);
+    // Verify IPv6 stack is down and Thread stack is not fully enabled.
```

Comment:
```suggestion
    // Verify IPv6 interface is up and Thread stack is enabled.
```
I am also fine to remove this comment since the `VerifyOrExit()` code below is clear.

Reply:
+1  on removing the comment fully (I also feel it's clear from the `VerifyOrExit()`)

Reply:
```suggestion
```